themes/default/index.twig

```diff
@@ -1,49 +1,70 @@
 <!DOCTYPE html>
-<html lang="en" class="no-js">
+<html lang="en">
 <head>
-    <meta charset="utf-8" />
-
-    <title>{% if meta.title %}{{ meta.title }} | {% endif %}{{ site_title }}</title>
+    <!-- Page Meta -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
     {% if meta.description %}
-        <meta name="description" content="{{ meta.description|striptags }}">
+    <meta name="description" content="{{ meta.description|striptags }}">
     {% endif %}{% if meta.robots %}
-        <meta name="robots" content="{{ meta.robots }}">
+    <meta name="robots" content="{{ meta.robots }}">
     {% endif %}
 
-    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,700" type="text/css" />
-    <link rel="stylesheet" href="{{ theme_url }}/style.css" type="text/css" />
+    <!-- Page Title -->
+    <title>{% if meta.title %}{{ meta.title }} | {% endif %}{{ site_title }}</title>
+
+    <!-- Fonts -->
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,700" type="text/css">
+
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="{{ base_url }}/vendor/twbs/bootstrap/dist/css/bootstrap.min.css" type="text/css" />
```

Comment:
Thanks for the fresh eyes :eyes:  @PhrozenByte !

Since 4.0.0 isn't available on CDN yet, I opted to just create a `post-install-cmd` script in `composer.json` to copy the needed files to the theme directory, and therefore keep utilizing composers depenodency-managemnt.